css-three-column-layout/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+  <div id="container">
+
+    <div>
+      <h2 class="text">
+        Designed for life science. optimized for SaMD and Digital Health.
+      </h2>
+    </div>
+
+    <div class="box">
+
+      <div class="row">
```

Comment:
You're not using rows correctly. All items that are next to each other in a horizontal line would be considered a row. So in this instance, this would technically be acting as a column. All the images would technically be considered 1 row, and then the text below it would be considered a new row